docs/synthetics/ci-cd/overview.md

```diff
@@ -10,16 +10,16 @@ If you're interested in using our API, you can find more information about it on
 
 Before we discuss the specifics of setting up the various integrations, you first need to collect some information specific to your Sematext Cloud account.
 
-### Finding your Synthetics App and Monitor IDs
+### Finding your monitor information
```

Comment:
Inconsistent casing with other titles?

Reply:
Did you mean that capitalizing `Monitors` would fit better? I checked the rest of Synthetics docs and I see the pattern of only capitalizing keywords in cases like:
- [**Configuring Custom Alerts based on JSON response**](https://sematext.com/docs/synthetics/http-monitor/#configuring-custom-alerts-based-on-json-response) - Only keywords capitalized
- [**Run environment**](https://sematext.com/docs/synthetics/browser-monitor/#run-environment) - Only keywords capitalized
- [**Restrict Access with Password Protection**](https://sematext.com/docs/synthetics/http-monitor/#bulk-adding-http-monitors) - Mixed casing, `Access` isn't consistent with others